src/stylesheets/_typography.scss

```diff
@@ -5,49 +5,42 @@
         font-size: 2.2rem;
     }
 }
-
 @mixin font-size-header-2 {
     font-size: 1.6rem;
 
     @include respond-to(md) {
         font-size: 1.8rem;
     }
 }
-
 @mixin font-size-text-1 {
     font-size: 1.4rem;
 
     @include respond-to(md) {
         font-size: 1.6rem;
     }
 }
-
 @mixin font-size-text-2 {
     font-size: 1.4rem;
 }
-
 @mixin font-size-text-3 {
     font-size: 1.2rem;
 }
 
 h2 {
     font-family: $font-primary;
-    font-weight: 500;
     line-height: 1.24;
     margin: 3.6rem 0 2.4rem;
-    font-weight: normal;
+    font-weight: 500;
```

Comment:
See, here we switch from `"normal"` to `500` and in the other place from `400` to `"normal"`. Yuck. Numbers should be used everywhere.